instrumentation/pg/lib/opentelemetry/instrumentation/pg/patches/connection.rb

```diff
@@ -84,7 +84,7 @@ def span_attrs(kind, *args) # rubocop:disable Metrics/AbcSize
             end
 
             attrs = { 'db.operation' => validated_operation(operation), 'db.postgresql.prepared_statement_name' => statement_name }
-            attrs['db.statement'] = sql unless config[:db_statement] == :omit
+            attrs['db.statement'] = sql.to_s if config[:db_statement] != :omit && !sql.nil? && sql.respond_to?(:to_s)
```

Comment:
Would safe navigation do the trick? 

```suggestion
            attrs['db.statement'] = sql.to_s if config[:db_statement] != :omit && sql&.to_s
```